pkg/kubelet/kubelet_node_status_test.go

```diff
@@ -86,8 +91,11 @@ func generateTestingImageList(count int) ([]kubecontainer.Image, []v1.ContainerI
 
 		expectedImageList = append(expectedImageList, apiImage)
 	}
-	// 3. only returns the top maxImagesInNodeStatus images in expectedImageList
-	return imageList, expectedImageList[0:maxImagesInNodeStatus]
+	// 3. only returns the top maxImages images in expectedImageList
+	if maxImages == 0 { // 0 means no limit
```

Comment:
Is `0 == unlimited` convention for k8s?  I seem to recall `-1 == unlimited` being more common, but you spend a lot more time around configuration than I do ;)

Reply:
I guess we shouldn't block reporting none, if someone really wants to do that? Easy to change to -1.

Reply:
PodPidsLimit has precedence for -1 being unlimited

Reply:
QPS parameters use 0

Reply:
Sounds good

Reply:
hmmm

Reply:
I am fine with either then.  Up to you

Reply:
Changed it to -1, which I think is more reasonable. 0 => limit to none is a natural thing to assume, IMO.